test/integration/thruster.cc

```diff
@@ -46,15 +46,18 @@ class ThrusterTest : public InternalFixture<::testing::Test>
   /// \param[in] _density Fluid density
   /// \param[in] _diameter Propeller diameter
   /// \param[in] _baseTol Base tolerance for most quantities
+  /// \param[in] _useAngVelCmd Send commands in angular velocity instead of
+  /// force
   public: void TestWorld(const std::string &_world,
       const std::string &_namespace, double _coefficient, double _density,
-      double _diameter, double _baseTol);
+      double _diameter, double _baseTol, bool _useAngVelCmd = false,
+      double mass = 100.1);
```

Comment:
Style nit 

```suggestion
      double _mass = 100.1);
```

Also below